src/execute-pr-actions.ts

```diff
@@ -106,6 +111,32 @@ function getMutationsForCommentRemovals(actions: Actions, botComments: ParsedCom
     });
 }
 
+function getMutationsForSuggestions(actions: Actions, pr: PR_repository_pullRequest) {
+    if (pr.reviews?.nodes?.find(review => review?.author?.login === "typescript-bot")) return [];
```

Comment:
Done :heavy_check_mark:

Reply:
Maybe I'm missing something, or maybe you didn't push more commits yet, but the code that I see doesn't do that.

But more than that, the code that I see posts a new review regardless of whether one exists: will the GH api update an existing one?  If not, then it will post a new review on every run.

Reply:
It excludes existing suggestions' diffs from subsequent suggestions, so `suggestions` will be `[]` if they were already posted. I should've added some comments, adding them now :heavy_check_mark: